app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -141,26 +155,60 @@ fun SyncedTabsList(
 }
 
 /**
- * Text header for sections of synced tabs
+ * A non-collapsible text header for sections of synced tabs
+ *
+ * @param sectionText The section title for a group of synced tabs.
+ */
+@Composable
+fun SyncedTabsSectionHeader(sectionText: String) {
+    SyncedTabsCollapsibleSectionHeader(sectionText = sectionText)
```

Comment:
The thought here was to have a single source for the Header UI, since they are the same UI minus the expand/collapse chevron. I'm open to thoughts though on whether we should have a Composable for each distinct deviation the UI can take (e.g. display-only header, clickable header, collapsible header, etc.)